ifrs17/Import/ImportStorage.ipynb

```diff
@@ -224,7 +224,10 @@
                 "\n        var openingRawVariables = Enumerable.Empty<RawVariable>();",
                 "\n        var openingIfrsVariables = Enumerable.Empty<IfrsVariable>();",
                 "\n",
-                "\n        if(allImportScopes.Select(dn => DataNodeDataBySystemName[dn]).Any(dnd => dnd.Year < args.Year)) {",
+                "\n        var allImportScopesAtInceptionYear = allImportScopes.Select(dn => DataNodeDataBySystemName[dn]).Where(dnd => dnd.Year == args.Year).Select(x => x.DataNode).ToArray();",
```

Comment:
make it an HashSet then Except is better performing

Reply:
Done.